packages/docs/docs/getting_started.md

```diff
@@ -12,6 +12,97 @@ If you are using Javascript, grab the package here:
 
 https://www.npmjs.com/package/@glasseaters/hydra-sdk
 
-::: warning
+:::warning
 Everything is brand new so expect frequent changes.
 :::
+
+## Quick Start - JS
+
+Install the package from npm:
+```bash
+yarn add @glasseaters/hydra-sdk
+```
+
+This is how you'd setup a Hydra with the Wallet [membership model](intro.md#adding-members).
+
+```ts
+import {Connection, Keypair, LAMPORTS_PER_SOL} from "@solana/web3.js";
+import {ASSOCIATED_TOKEN_PROGRAM_ID, Token, TOKEN_PROGRAM_ID} from "@solana/spl-token";
+import {airdrop} from "@metaplex-foundation/amman";
+import {
+    Fanout,
+    FanoutClient,
+    FanoutMembershipMintVoucher,
+    FanoutMembershipVoucher,
+    FanoutMint,
+    MembershipModel
+} from "@glasseaters/hydra-sdk";
+
+
+const connection = new Connection("devnet", "confirmed");
+const authorityWallet = Keypair.generate();
+
+await airdrop(connection, authorityWallet.publicKey, LAMPORTS_PER_SOL * 2);
+
+const fanoutSdk = new FanoutClient(
+    connection,
+    new NodeWallet(new Account(authorityWallet.secretKey))
+);
+
+// Initialize the Hydra Wallet
+const {fanout} = await fanoutSdk.initializeFanout({
+    totalShares: 100,
+    name: `Your Globally Unique Wallet Name`,
+    membershipModel: MembershipModel.Wallet,
+});
+
+// Retrieve the On-chain Hydra Wallet
+const fanoutAccount = await fanoutSdk.fetch<Fanout>(
+    fanout,
+    Fanout
+);
+
+// This is your Hydra Wallet Address
+const HydraAccountKey = fanoutAccount.accountKey
```

Comment:
@austbot - What's the recommended way to access the Hydra Wallet's public key once it's on chain?

Reply:
That makes a lot of sense (it's fast because the PDA doesn't require a trip to the chain). Where do you get the `bump` in that example.